src/components/WalletModal/Option.test.tsx

```diff
@@ -16,25 +16,25 @@ beforeEach(() => {
   mocked(useAccountDrawer).mockReturnValue([true, mockToggleDrawer])
 })
 
-const mockConnection1: Connection = {
+const mockConnection1 = {
   getName: () => 'Mock Connection 1',
+  activate: jest.fn(),
   connector: {
-    activate: jest.fn(),
     deactivate: jest.fn(),
   } as unknown as Connector,
   getIcon: () => UNIWALLET_ICON,
   type: ConnectionType.UNISWAP_WALLET,
-} as unknown as Connection
+} as unknown as Connection & { connector: Connector }
```

Comment:
can you explain this change?

Reply:
In order to mock, we cannot use an optional operator, so we need to assert that this type is not undefined. Rather than use an `assert` I just define the type in it's declaration.